docs/monitoring/03-02-istio.md

```diff
@@ -14,3 +14,28 @@ The Istio-related instance is a Deployment named `monitoring-prometheus-istio-se
 See the diagram for a broader view of how the Istio-related instance fits into the monitoring setup in Kyma:
 
 ![Istio Monitoring](./assets/monitoring-istio.svg)
+
+By default `minitoring-prometheus-istio-server` is not provided as a datasource in grafana. However, this can be enabled by adding following override 
```

Comment:
```suggestion
By default, `minitoring-prometheus-istio-server` is not provided as a data source in Grafana. However, this can be enabled by adding the override: 
```